llama-index-integrations/program/llama-index-program-openai/llama_index/program/openai/base.py

```diff
@@ -1,6 +1,7 @@
 import logging
 from typing import Any, Dict, Generator, List, Optional, Tuple, Type, Union, cast
 
+from pydantic import ValidationError
```

Comment:
I wonder if we'll run into problems here if we don't import this from `llama_index.core.bridge.pydantic` instead.